src/amo/reducers/users.js

```diff
@@ -152,15 +155,17 @@ type EditUserAccountAction = {|
 |};
 
 export const editUserAccount = ({
-  errorHandlerId, picture, userFields, userId,
+  errorHandlerId, notifications, picture, userFields, userId,
 }: EditUserAccountParams): EditUserAccountAction => {
   invariant(errorHandlerId, 'errorHandlerId is required');
+  invariant(notifications, 'notifications are required');
+  invariant(picture !== undefined, 'picture is required');
```

Comment:
Why do we need to check explicitly for `picture !== undefined`?

Reply:
it can be null